fcrepo-kernel-impl/pom.xml

```diff
@@ -119,6 +119,12 @@
       <version>${project.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
`fcrepo-jms` also uses awaitility, so maybe we should move the versioning info up into `fcrepo4`'s pom?